src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -107,7 +98,7 @@ function status()
     fi
 
     echo
-    status_out=`sudo -u "${SUDO_USER}" launchctl list | grep "${SVC_NAME}"`
+    status_out=`launchctl list | grep "${SVC_NAME}"`
```

Comment:
It's because $() can be nested.  I'll change
http://stackoverflow.com/questions/4708549/whats-the-difference-between-command-and-command-in-shell-programming